packages/react-storage/src/components/StorageBrowser/Controller.tsx

```diff
@@ -10,7 +10,7 @@ export function Controller(): null {
   const [, handleListLocations] = useLocationsData();
 
   React.useEffect(() => {
-    handleListLocations({ options: { pageSize: 1000, refresh: true } });
```

Comment:
Added default fallback pageSize to `listLocationsAction.ts`

Reply:
It's intentionally handled this way and will be updated with pagination

Reply:
ahh got it, thanks reverted